app/assets/stylesheets/legacy_sass/sayt/custom.sass

```diff
@@ -52,6 +52,14 @@
     opacity: 0
     filter: Alpha(Opacity = 0)
 
+/* Hiding 'No search results.' text 
+ *----------------------------------
+ 
+#usasearch_sayt ~ .ui-helper-hidden-accessible
+  position: absolute !important
```

Comment:
what's the difference with something with `display: none`?

Reply:
it's used for accessibility so people with screen readers can read it. By using display: none; you hide it from screen readers as well. So, it's better to move it off the screen with position: absolute;